pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "benchmark"
+version = "0.1.0"
+description = ""
+authors = ["Daniel Kleczynski <danielkleczynski@gmial.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+python-dotenv = "^1.0.0"
+aiohttp = {extras = ["speedups"], version = "^3.9.0"}
+pydantic = "^2.5.1"
+langchain = "^0.0.329"
+openai = "^1.0.0rc3"
+pysqlite3 = "^0.5.0"
```

Comment:
**suggestion(non-blocking):** Python has a built-in support for sqlite3: https://docs.python.org/3/library/sqlite3.html